recipes/vtk/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+import os
+from conans import ConanFile, CMake, tools, RunEnvironment
+
+class TestVTKConan(ConanFile):
+    settings = "os", "compiler", "arch", "build_type"
+    generators = "cmake" # Replace "cmake" by "cmake_find_package_multi" to test multi
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self.settings):
+            # Replace "bin" by "Debug" on Win or by "." (dot) on Mac when testing "cmake_find_package_multi"
+            bin_path = os.path.join("bin", "test_vtk_package")
+            # Lnx only. Remove me when Lnx succeed: self.run("LD_LIBRARY_PATH=%s %s" % (os.environ.get('LD_LIBRARY_PATH', ''), bin_path), run_environment=True)
```

Comment:
The `run_environment=True` argument does make doing `LD_LIBRARY_PATH=...` superfluous.
```suggestion
```

Reply:
Done